gluoncv/model_zoo/resnext.py

```diff
@@ -328,6 +362,36 @@ def resnext101_64x4d(**kwargs):
     kwargs['use_se'] = False
     return get_resnext(101, 64, 4, **kwargs)
 
+
+def resnext101b_64x4d(**kwargs):
```

Comment:
maybe we name it as d, to match our resnet naming.

Reply:
it is actually e variant i think

Reply:
then probably name it to e? otherwise users won't understand the naming system.